.gitignore

```diff
@@ -0,0 +1,11 @@
+dist/
+node_modules/
+coverage/
+.vscode/
+.env
+
+.DS_Store
```

Comment:
Minor, but I really think we shouldn't include OS- or editor-specific entries in projects' gitignores. Those should be in each user's global gitignore.

Reply:
ah sorry, copypasta from j1 node client

Reply:
will remove later so that dad can do travis things